localization/english/wc_traits_l_english.yml

```
@@ -29,6 +29,9 @@
  trait_creature_high_elf:0 "High Elf"
  trait_creature_high_elf_desc:0 "The high elves, or Quel'dorei, are the absolute opposite of their cousins: the night elves. The Quel'dorei are the descendants of those who served Azshara at the time of the first Well of Eternity. The high elves used magic for generations, absorbing it with their mothers' milk. Even among those lacking a natural talent for magic, high elves cannot live or even think clearly if they do not receive their necessary mana requirement on a regular basis."
  trait_creature_high_elf_character_desc:0 "$trait_creature_high_elf_desc$"
+ trait_creature_void_elf:0 "Void Elf"
+ trait_creature_void_elf_desc:0 "Void elves are a sect of former blood elves that infused themselves with the powers of the Void for the sake of defending Azeroth, all the while resisting the Void's call to madness. In Thalassian, they are called #italic ren'dorei#!, which means "children of the Void"."
```

Capitalized the "Children of the Void". I also agree with @TheDrPibb1999 and that the phrase "for the sake of defending Azeroth" does not really fit as Void Elf description. It is not really explicitly mentioned in the quest cited by Wowpedia (https://wowpedia.fandom.com/wiki/The_Void_Elves_Stand_Ready) and more of a general theme during BfA that they are defending Azeroth. In GoA setting this does not really fit as they might have different agendas.
```suggestion
 trait_creature_void_elf_desc:0 "Void elves are a sect of former blood elves that infused themselves with the powers of the Void, all the while resisting the Void's call to madness. In Thalassian, they are called #italic ren'dorei#!, which means "Children of the Void"."
```

---

Changed this in f3dda92d467c9ed2c7533a1de5071c9db081ec94.